pkg/controllers/management/node/utils.go

```diff
@@ -41,14 +41,14 @@ const (
 
 func buildAgentCommand(node *v3.Node, dockerRun string) []string {
 	drun := strings.Fields(dockerRun)
-	cmd := []string{"--native-ssh", "ssh", node.Spec.RequestedHostname}
+	cmd := []string{"ssh", node.Spec.RequestedHostname}
```

Comment:
note to reviewers:
by default, rancher-machine uses the external ssh binary found in the running environment.